### PR TITLE
[TASK] Explicitly include .htaccess into the rsync list

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Comment by helhum** [comment]
Can you elaborate why you need this at all?
Shouldn't all files be rsynced by default

File: recipe/helhum/typo3.php [hunk 1]
```diff
@@ -140,6 +140,7 @@
         '/var/log',
     ],
     'include' => [
+        '/.htaccess',
         '/var/log/.gitkeep',
```
Comment (helhum) [inline recipe/helhum/typo3.php:144]: unrelated, but I think, since var/log now is shared, we can remove this here

**Comment by oliverklee** [comment]
When I used this recipe, the .htaccess in the root directory did not get synced. This was with an older version of the recipe, though. I'll retest with the current version.

**Comment by helhum** [comment]
> This was with an older version of the recipe

Yeah, could be with the older version, where I tried to use rsync filters, which turned out to be too inflexible and hard to understand

File: recipe/helhum/typo3.php (hunk 1 above)
Comment (helhum) [reply recipe/helhum/typo3.php:144]: See #7

**Comment by oliverklee** [comment]
I've just tested this with the current version. This change indeed is not needed anymore. Hence closing this PR.

**Pull request closed without merging**